lib/rucio/common/didtype.py

```diff
@@ -126,15 +127,18 @@ def _did_from_str(self, did: str) -> None:
         Construct the DID from a string.
         :param did: string containing the DID information
         """
-        did_parts = did.split(DID.SCOPE_SEPARATOR, 1)
-        if len(did_parts) == 1:
-            self.name = did
-            self._update_implicit_scope()
-            if not self.has_scope():
-                raise DIDError('Object construction from non-splitable string is ambigious')
-        else:
-            self.scope = did_parts[0]
-            self.name = did_parts[1]
+        try:
+            self.scope, self.name = extract_scope(did)
+        except (ImportError, InvalidAlgorithmName, ConfigNotFound, NoSectionError):  # Only use when the policy can not be found
+            did_parts = did.split(DID.SCOPE_SEPARATOR, 1)
```

Comment:
We could add visibility into when fallback happens:

```suggestion
             logging.debug(f"extract_scope fallback for '{did}': {type(e).__name__}")
            did_parts = did.split(DID.SCOPE_SEPARATOR, 1)
```